docs/references/errors.md

```diff
@@ -51,57 +51,58 @@ message GreetingRequest {
 }
 ```
 
-## META0003
+## META0003 {#META0003}
 
-Cannot reach the endpoint to execute service discovery. Make sure:
+Cannot reach the deployment to execute service discovery. Make sure:
 
-* The provided `uri` is correct
-* The service endpoint is up and running
-* The Restate runtime can reach the service endpoint through the configured `uri`
+* The provided `uri`/`arn` is correct
```

Comment:
```suggestion
* The provided `URI`/`ARN` is correct
```
Also for the other occurences of URI and ARN on this page.

Reply:
Will fix it in the restate repo.